Jasmine/Tetris/ViewModels/TetrisTileView.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+/// Represents the view for a tile
+class TetrisTileView: UILabel {
```

Comment:
i'm not concerned with the physical folder struct atm, can sort it out later with the 3rd party tool. 
It's in VC in XCode's folder structure